### ISSUE TITLE
[A11Y] amp-image-lightbox: `aria-describedby` should be set to the caption element when available.

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#5502]
When `amp-image-lightbox` discovers the caption for an image, it created a `div` with `amp-image-lightbox-caption -amp-image-lightbox-caption` as the class. It also needs to set the `aria-describedby` on the cloned `img` to the id of this caption div.

**Comment by aghassemi** [comment]
@zhouyx, very related to #5501, so you get two issues!
